tcms/kiwi_auth/tests/test_views.py

```diff
@@ -199,6 +200,24 @@ def test_register_user_and_activate_by_admin(self):
                                 '<a href="mailto:{}">{}</a>'.format(email, name),
                                 html=True)
 
+    def test_invalid_form(self):
+        response = self.client.post(
+            self.register_url,
+            {
+                "username": "kiwi-tester",
+                "password1": "password-1",
+                "password2": "password-2",
+                "email": "new-tester@example.com",
+            },
+            follow=False,
+        )
+
+        self.assertContains(
+            response, 'The two password fields didn’t match.'
+        )
```

Comment:
Tests in French are failing b/c this error message appears translated to the user when running in a different locale. 
https://github.com/kiwitcms/Kiwi/pull/2143/checks?check_run_id=1537516256

When asserting on it you should assert on the translation. See how other tests are doing it.